doc-botonic/docs/concepts/18.webchat.md

```diff
@@ -30,6 +30,156 @@ Botonic supports the following browsers:
 | Internet Explorer | 11 (with polyfills) |
 
 
+
+### How to make your bot compatible with IE11
+
+To make a bot compatible with IE11, its javascript code must be compliant with ES5. To do so, babel is used to transpile all the code needed to ES5.
+
+#### Prepare the bot
+
+To make the bot work in IE11:
+
+1. Add the following meta in the bot's integration code:
+
+```javascript 
+<meta http-equiv="X-UA-Compatible" content="IE=11" />
+```
+
+2. If style rendering issues occur, [disable webchat animations](/templates/template-custom-webchat):
+
+```javascript 
+theme: {
+  animations: {
+    enable: false,
+  },
+}
+```
+
+#### Add polyfills
+
+Some functionalities supported by most browsers are not implemented in IE11. To use them, polyfills are needed. 
+
+Here is a list of polyfills used in some of the bots:
+
+- Basic polyfills: [core-js](https://www.npmjs.com/package/core-js)
+- Style issues: [@webcomponents/webcomponentsjs](https://www.npmjs.com/package/@webcomponents/webcomponentsjs)
+- scrollTo feature: [element-scroll-polyfill](https://www.npmjs.com/package/element-scroll-polyfill)
+- normalize: [unorm](https://www.npmjs.com/package/unorm)
+
+To add polyfills:
+
+1. Open the `webpack.config.js` file.
+
+2. Add all the polyfills that are currently used in the entry property within the `botonicWebchatConfig`. The first 2 polyfills are mandatory, as well as the 2 that follows:
```

Comment:
" The first 2 polyfills are mandatory, as well as the 2 that follows" 
I don't understand if it refers to the 2 polyfills from above
"the 2 that follows" not clear which ones it refers to

Reply:
Yes, maybe we should say explicitly which ones are the first 2 and the 2 that follows. Maybe this text is more explanatory:
"The first 2 polyfills are mandatory (`core-js/stable` and `regenerator-runtime/runtime`), as well as the 2 that follows (`@webcomponents/webcomponentsjs` and `element-scroll-polyfill`)"
What do you think @dpinol @annehubtype ?

Reply:
@annehubtype for all the bots that I've made compatible with IE11 so far I've needed these 2 polyfills (`@webcomponents/webcomponentsjs` and `element-scroll-polyfill`), so maybe it will be better to keep it the way it is now, with the 4 polyfills mandatory. 
The last 2 are used for styles (`@webcomponents/webcomponentsjs`) and scrollTo feature (`element-scroll-polyfill`) and both are used by default in @botonic/react.